setup.cfg

```diff
@@ -0,0 +1,52 @@
+[metadata]
+name = cohortfinder
+url = https://github.com/choosehappy/CohortFinder
+download_url = https://github.com/choosehappy/CohortFinder
+license = BSD
+license_file = LICENSE.txt
+description = HistoQC is an open-source quality control tool for digital pathology slides
+long_description = file: Readme.md
+long_description_content_type = text/markdown
+author = Andrew Janowczyk
+classifiers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Visualization
+    Topic :: Scientific/Engineering :: Information Analysis
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Topic :: Utilities
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: MacOS
+
+
+[options]
+packages =
+    cohortfinder
+python_requires = >=3.7
+install_requires =
+    matplotlib==3.3.4
+    numpy==1.20.1
+    opencv_python_headless==4.5.3.56
+    pandas==1.3.0
+    scikit_learn==1.1.1
+    scipy==1.6.1
+    umap==0.1.1
```

Comment:
I am not sure, maybe @jacksonjacobs1 Jackson can comment on this?

Reply:
As Andrew previously commented, this should be umap-learn: https://pypi.org/project/umap-learn/

I think I didn't catch this because umap-learn was already installed in my python environment, so when python imported `umap` the correct module was found and imported without an error.

Reply:
Thanks, push a commit with the following modification:
1. For both readme file and setup.cfg, delete umap and use umap_learn instead.
2. In the previous version of setup.cfg, there is an error for the scikit-learn version, I updated it with the version in my environment.
> ERROR: No matching distribution found for scikit-learn==1.1.1 (from cohortfinder==0.0.post38+g0bea166.d20240214)